localization/languages/pt-BR.json

```diff
@@ -224,7 +224,7 @@
         "loginPromptTitle": null, //missing translation
```

Comment:
I didn't quite understand that part, could you explain it to me

Reply:
The original string is this:
```
        "loginPromptTitle": "Sign in to %h (%r)", //%h is replaced with host, %r with realm (title of protected part of site)
```

It's displayed when signing into sites that use HTTP authentication:
<img width="653" alt="Screen Shot 2020-04-24 at 6 33 17 PM" src="https://user-images.githubusercontent.com/10314059/80264536-1a9e9300-865a-11ea-939e-2b943dd281c5.png">

Reply:
I still don't know how to make it friendly, for now it can be (log into or log in -> **Logar em**) , what do you think?, does it make sense to you?

Reply:
I don't know how to answer the grammar question, but I guess it would be the same as what other login forms do. If you go here: https://github.com/login, there's a heading that says "sign in to GitHub" - how is that translated?